.github/workflows/task-test.yml

```diff
@@ -177,12 +177,13 @@ jobs:
                              ${{ format('{0}', inputs.coverage && 'coverage test' || '') }} \
                              ${{ inputs.container || inputs.env }} ${{ runner.arch }} - Redis ${{ inputs.get-redis || 'unstable' }}" | \
                        sed -e 's/[":\/\\<>\|*?]/_/g' -e 's/__*/_/g' -e 's/^_//' -e 's/_$//')" >> $GITHUB_OUTPUT
+      - name: Enable Intel's Optimizations
+        run: echo "BUILD_INTEL_SVS_OPT=yes" >> $GITHUB_ENV
       - name: Build
         env:
           SAN: ${{ inputs.san }}
           REDIS_VER: ${{ inputs.get-redis }}
           ENABLE_ASSERT: 1
-          BUILD_INTEL_SVS_OPT: 1
```

Comment:
It would be better to have it here, or at the global `env`. It wasn't picked up? Maybe we missed something in the makefile?

Reply:
It was only valid for this step, while needed also for testing.